src/pages/Home/Home.styled.jsx

```diff
@@ -51,7 +58,7 @@ export const ManImage = styled.img`
   object-fit: contain;
   position: absolute;
   transform: translateY(-80%);
-
+  max-height: 350px;
   position: relative;
```

Comment:
Position is repeated, can you check which one is correct and erase the other one, please?